openslides_backend/action/generics.py

```diff
@@ -73,10 +73,6 @@ def create_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
                     )
                     for instance_field, replacement in structured_fields:
                         if isinstance(field, BaseTemplateRelationField):
-                            if not ID_PATTERN.match(replacement):
-                                raise ActionException(
-                                    "Template relation fields can only use replacements which are ids."
-                                )
```

Comment:
This was wrong: e.g. `logo_$_id` is a template relation field which can hold non-id replacements. If we want to differentiate between this and "normal" template relation fields, we would have to add a `TemplateRelationTagField` (or something like this). @normanjaeckel if you want, feel free to do this, but I think for now this suffices. We just have to care to only write valid data to the fields in the actions, but we have to care about that anyway.